util/cc2file.go.tmpl

```diff
@@ -0,0 +1,34 @@
+{{/*
+	This command sends your CC(s) code in a text file, rather than "plain" Discord messages, preserving Tabs, markdown, etc. 
+	You can especify more than one ID or trigger, and bot will generate one file per input parameter. If none are provided, output will be CC list
+	
+	Usage: `-cc2file <ID or Trigger> [ID or Trigger] [ID or Trigger]...`. For example: `-cc2file 10 "te st" 23`
+	Recommended trigger: Command trigger with trigger `cc2file`.
+	Configuration variables: None
+*/}}
+
+
+{{range .CmdArgs}}
```

Comment:
You need to ensure that you don't overflow the limit for `exec` calls.

Reply:
That's true. So, checking for `len .CmdArgs` will do the trick
Another pending issue to solve. Thanks!

Reply:
Another option I see is `split .CmdArgs 0 99` and `range` over that. And warn user if (s)he asked for more than 100 CCs

Reply:
Done! Now max limit is 15 per whole execution, and if there are still args, bot will say "run again this CC with `-cc2file <list of remaining args>`"